tools/coqmod/dune

```diff
@@ -0,0 +1,11 @@
+(executable
```

Comment:
you should add `(vendored_dirs csexp)` to avoid formatting the vendored code and ignoring its warnings.